.ci/ci_cache_qemu.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+source "${cidir}/../lib/common.bash"
+source "${cidir}/lib.sh"
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+CURRENT_QEMU_VERSION=$(get_version "assets.hypervisor.qemu.version")
```

Comment:
Random thought: could this script be made generic so that it would work for other hypervisors (nemu, etc)?

Reply:
yeah, I think we can refactor some of this code. Opened #1782 to address it.

Reply:
Thanks @chavafg!